lighthouse-core/test/lib/minification-estimator-test.js

```diff
@@ -9,8 +9,9 @@ const fs = require('fs');
 const assert = require('assert').strict;
 const {computeCSSTokenLength, computeJSTokenLength} = require('../../lib/minification-estimator.js'); // eslint-disable-line max-len
 
-const angularFullScript = fs.readFileSync(require.resolve('angular/angular.js'), 'utf8');
-
+const angularJs = fs.readFileSync(require.resolve('angular/angular.js'), 'utf8');
+const courseheroFilename = `${__dirname}/../../test/fixtures/source-maps/coursehero-bundle-2.js`;
+const courseheroBundle = fs.readFileSync(courseheroFilename, 'utf8');
```

Comment:
angularJs, courseheroBundle-both are strings of JS. can they have the same naming format?